src/commands/utils/eval.js

```diff
@@ -0,0 +1,43 @@
+const Command = require('../../classes/Command')
+
+module.exports = new Command((msg, suffix) => {
+  const util = require('util')
+  try {
+    const returned = eval(suffix) // eslint-disable-line no-eval
+    let str = util.inspect(returned, {
+      depth: 1
+    })
+    if (str.length > 1900) {
+      str = str.substr(0, 1897)
+      str = str + '...'
+    }
+    str = str.replace(new RegExp(process.env.BOT_TOKEN, 'gi'), '( ͡° ͜ʖ ͡°)')
```

Comment:
Lennyface is so last season lol, should we put something else here like
```suggestion
    str = str.replace(new RegExp(process.env.BOT_TOKEN, 'gi'), '<no token leeks pls kthx>')
```